notebooks/examples/quickstart_in_5_minutes.ipynb

```diff
@@ -164,14 +164,14 @@
      "name": "stdout",
```

Comment:
<p>Navigation notes:</p><p> </p><ol><li>Checks that have no display data - their link doesn't do anything, please fix (either no link or linking to them in the bottom table of "Other Checks That Weren't Displayed). </li><li>Link back to the summary: please change the location to be at the bottom of the check's output and the wording to 'Go to top' (as I think 'Go to summary' isn't very accurate - some checks doen't have a summary...)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
it's back to the suite summary

Reply:
about 1 I'll think what I can do, maybe it's ok to leave it like that because otherwise the table will look funny

Reply:
About 1 - I agree that it won't look funny very nice if only some have links, unless we reorder them to the bottom in which case it can look ok IMO. Anyway I'm open to other suggestions as well, but not to leaving it like it is currently (bad UX experience for the user, it feels like a bug - dysfunctional link)

About 2 - I know it's back to the suite's condition summary, that's fine. I related to the location of the link (in relation to the check's output. Currently it's between the title and the description, not at the end of the output of each check) and to the wording of the link.

Reply:
I agree with @shir22 about 1. - We should reorder so that checks with links will appear at the top of the table.